src/views/CountryNsOverviewContextAndStructure/NsDirectoryInitiatives/InitiativeCard/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { _cs } from '@togglecorp/fujs';
+
+import Container from '#components/Container';
+import TextOutput from '#components/TextOutput';
+import useTranslation from '#hooks/useTranslation';
+
+import styles from './styles.module.css';
+import i18n from './i18n.json';
+
+interface Props {
+    className?: string;
+    id: number;
+    allocation: number | null | undefined;
+    categories: string[] | null | undefined;
+    fund_type: string | null | undefined;
+    funding_period: number | null | undefined;
+    title: string | null | undefined;
+    year: string | null | undefined;
```

Comment:
Instead of defining types as this, we could use type information from the generated types.